### PR TITLE
Adds flag to output application logs in JSON format

### LINKED ISSUE #1494: Implement ApplicationLogJSONEnabled for ApplicationLog json formatted log lines
**Is your feature request related to a problem? Please describe.**
We log everything in JSON and for some logs such as the ApplicationLog it does not seem to result in JSON.
I am not sure if this is a configuration error or not but I don't see it.

**Describe the solution you would like**
A setting called ApplicationLogJSONEnabled to output JSON

**Describe alternatives you've considered (optional)**
Basically pipelines or filtering to turn it into JSON (outside of skipper).

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Fixes #1494

Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: logging/log.go [hunk 1]
```diff
@@ -45,14 +48,15 @@ func (f *prefixFormatter) Format(e *logrus.Entry) ([]byte, error) {
 	return append([]byte(f.prefix), b...), nil
 }
 
-func initApplicationLog(prefix string, output io.Writer) {
-	if prefix != "" {
-		logrus.SetFormatter(&prefixFormatter{
-			prefix, logrus.StandardLogger().Formatter})
+func initApplicationLog(o Options) {
```
Comment (AlexanderYastrebov) [inline logging/log.go:51]: Here is an example output of `bin/skipper -application-log-json-enabled`:
```json
{"level":"info","msg":"Expose metrics in codahale format","time":"2020-08-05T11:54:50+02:00"}
{"level":"warning","msg":"no route source specified","time":"2020-08-05T11:54:50+02:00"}
{"level":"info","msg":"support listener on :9911","time":"2020-08-05T11:54:50+02:00"}
{"level":"info","msg":"proxy listener on :9090","time":"2020-08-05T11:54:50+02:00"}
{"level":"info","msg":"TLS settings not found, defaulting to HTTP","time":"2020-08-05T11:54:50+02:00"}
```

There are a couple of things to clarify/polish:
* Is it ok to ignore `ApplicationLogPrefix` when JSON format is enabled?
* Is it ok that log level is lowercase?
* Access log uses custom datetime format presumably to match apache's log format. Is it ok to output timestamps in default logrus' format (`time.RFC3339`)?
* Access log uses customs time field name (`timestamp`). Is it ok to use default logrus' name (`time`)?

**Comment by AlexanderYastrebov** [comment]
@tachang @szuecs Please have a look

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**